streamlit_extras/badges/__init__.py

```diff
@@ -68,12 +79,15 @@ def example_streamlit():
 
 def example_github():
     badge(type="github", name="streamlit/streamlit")
+    
+def example_twitter():
+    badge(type="twitter", name="streamlit")
 
 
 __func__ = badge
 __title__ = "Badges"
-__desc__ = "Create custom badges (e.g. PyPI, Streamlit Cloud, GitHub)"
+__desc__ = "Create custom badges (e.g. PyPI, Streamlit Cloud, GitHub, Twitter)"
 __icon__ = "🏷️"
-__examples__ = [example_pypi, example_streamlit, example_github]
+__examples__ = [example_pypi, example_streamlit, example_github, example_twitter]
 __author__ = "Arnaud Miribel"
```

Comment:
This is missing! 😄 

```suggestion
__author__ = "Arnaud Miribel, ShruAgarwal"
```

Reply:
Thanks for adding me as an author 🤗. Glad it works fine. I had fun contributing here! 🧡